### PR TITLE
Tweaked Inner Focus

### PR DESCRIPTION
**Pull request by LOuroboros**
## Description
Addresses the missing piece of #1438 where the Ability PopUp would be casted incorrectly in the BattleScript triggered by Inner Focus...
**EDIT**: By removing it, because the official games don't call an Ability PopUp for Inner Focus :joy:

It'd be pointless to show a comparison, because there is no visual cue indicating the ability's effect, so please do test it locally.

## **Discord contact info**
Lunos#4026

### CODE REVIEW

File: data/battle_scripts_1.s [hunk 1]
```diff
@@ -7058,6 +7058,7 @@ BattleScript_ObliviousPreventsAttraction::
 
 BattleScript_FlinchPrevention::
```
Comment (ultima-soul) [inline data/battle_scripts_1.s:7059]: Since Inner Focus doesn't have a pop-up or text when it blocks certain Flinch effects (for context, it only says Inner Focus activated when a move has Flinching as its primary effect or if the flinch effect is certain/has 100% chance like with Fake Out), this battle script can be removed from here and `include/battle_scripts.h`.

File: src/battle_script_commands.c [hunk 2]
```diff
@@ -2739,18 +2739,9 @@ void SetMoveEffect(bool32 primary, u32 certain)
                 }
                 break;
             case MOVE_EFFECT_FLINCH:
-                if (GetBattlerAbility(gEffectBattler) == ABILITY_INNER_FOCUS)
+                if (gBattleMons[gEffectBattler].ability == ABILITY_INNER_FOCUS)
```
Comment (ultima-soul) [inline src/battle_script_commands.c:2742]: Why was this changed?

**Review by ultima-soul — CHANGES_REQUESTED** [review]

File: src/battle_script_commands.c (hunk 2 above)
Comment (LOuroboros) [reply src/battle_script_commands.c:2742]: Honestly, I don't remember why I changed that at all 😂 
I'll put it back to normal and test it out :eyes:

**Comment by LOuroboros** [comment]
Ready for re-review :eyes:

**Review by ultima-soul — APPROVED** [review]
